src/api.js

```diff
@@ -26,4 +26,15 @@ function getArticleById(article_id) {
     });
 }
 
-export { getArticles, getArticleById };
+function fetchCommentsForArticle(article_id) {
+  return api
+    .get(`/articles/${article_id}/comments`)
+    .then(({ data }) => {
+      return data.comments;
+    })
+    .catch((err) => {
```

Comment:
Catching the error too eerly here - you need to catch it in the component so you can display an error message if needed.